docker/mysql/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# This Dockerfile contains MySQL server and example datasets to be
+# populated into the server.  To COPY .sql dataset files outside of
+# this directory into the image, we need to run docker build from the
+# root directory of the SQLFlow source tree.
+#
+# To build this image:
+#
+#  cd $(git rev-parse --show-toplevel)
+#  docker build -t sqlflow:mysql -f docker/mysql/Dockerfile .
+#
+# To start a container executing this image:
+#
+#  docker run --rm -d -P --name mysql_server -v $PWD:/notify sqlflow:mysql
+#
+# To start a client container that can access this image:
+#
+#  docker run --rm --net=container:mysql_server -v $PWD:/notify sqlflow:ci
+#
+# The bind mount of $PWD on the host to /notify allows the
+# sqlflow:mysql container to create a file in the directory after
+# populating datasets, and the container running sqlflow:ci can use
+# inotifywait to wait for the creation of this file before it go on
+# running tests.
+FROM ubuntu:18.04
+
+COPY docker/mysql/install-mysql-server.bash /
+RUN /install-mysql-server.bash
```

Comment:
Maybe we can just copy the contents of `install-mysql-server.bash` as "Flat is better than nested".